odxtools/basecomparam.py

```diff
@@ -49,12 +50,13 @@ def __init_from_et__(self, et_element: ElementTree.Element,
             self.cptype = cast(StandardizationLevel, None)
             odxraise(f"Encountered unknown CPTYPE '{cptype_str}'")
 
-        cpusage_str = odxrequire(et_element.attrib.get("CPUSAGE"))
-        try:
-            self.cpusage = Usage(cpusage_str)
-        except ValueError:
-            self.cpusage = cast(Usage, None)
-            odxraise(f"Encountered unknown CPUSAGE '{cpusage_str}'")
+        cpusage_str = et_element.attrib.get("CPUSAGE")
```

Comment:
please add a comment, that the attribute is mandatory in ODX 2.2 and optional in 2.0, so that this will not be accidentially reverted in the future...

Reply:
```suggestion
        # Required in ODX 2.2, missing in ODX 2.0
        cpusage_str = et_element.attrib.get("CPUSAGE")
```